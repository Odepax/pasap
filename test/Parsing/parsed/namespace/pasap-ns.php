<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<pasap>
			<writing:util:md/>
			<:writing:util:md/>
			<util:md/>
			<:util:md/>
			<md/>
			<:md/>
		</pasap>
		<pasap pasap:ns="writing">
			<writing:util:md/>
			<:writing:util:md/>
			<util:md/>
			<:util:md/>
			<md/>
			<:md/>
		</pasap>
		<pasap pasap:ns="number">
			<one/>
			<number:two/>
			<three/>
			<one>
				<two>
					<three>
						<span>END</span>
					</three>
				</two>
			</one>
			<three>
				<pasap pasap:ns="writing">
					<writing:util:md/>
					<:writing:util:md/>
					<util:md/>
					<:util:md/>
					<md/>
					<:md/>
				</pasap>
			</three>
		</pasap>
	</body>
</html>
