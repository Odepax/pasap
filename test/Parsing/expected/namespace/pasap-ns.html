<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<pre>/empty/writing/util/md</pre>
		<pre>/empty/writing/util/md</pre>
		<pre>/empty/util/md</pre>
		<pre>/empty/util/md</pre>
		<pre>/empty/md</pre>
		<pre>/empty/md</pre>

		<pre>/empty/writing/writing/util/md</pre>
		<pre>/empty/writing/util/md</pre>
		<pre>/empty/writing/util/md</pre>
		<pre>/empty/util/md</pre>
		<pre>/empty/writing/md</pre>
		<pre>/empty/md</pre>

		<h1>One</h1>
		<h2>Two</h2>
		<h3>Three</h3>
		<h1>One</h1>
		<h2>Two</h2>
		<h3>Three</h3>
		<span>END</span>
		<h3>Three</h3>
		<pre>/empty/writing/writing/util/md</pre>
		<pre>/empty/writing/util/md</pre>
		<pre>/empty/writing/util/md</pre>
		<pre>/empty/util/md</pre>
		<pre>/empty/writing/md</pre>
		<pre>/empty/md</pre>
	</body>
</html>
